### PR TITLE
Update Makefile and README

### PR DESCRIPTION
**Pull request by peterbourgon**
- Build: make docker, make inplace, make (scope_ui_build.tar,static)
- Run: manually and in a container
- Rebuild static.go

Addresses #72

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,40 +12,39 @@ SCOPE_EXPORT=scope.tar
...
+
 $(PROBE_EXE): probe/*.go report/*.go xfer/*.go
 
 $(APP_EXE) $(PROBE_EXE):
 	go get -tags netgo ./$(@D)
 	go build -o $@ ./$(@D)
 
-$(FIXPROBE_EXE):
-	cd experimental/fixprobe && go build
+inplace: $(APP_EXE) $(PROBE_EXE)
```
Comment (tomwilkie) [inline Makefile]: Is this the only change?  Moving stuff around makes it hard to see.

File: Makefile [hunk 2]
```diff
@@ -12,40 +12,39 @@ SCOPE_EXPORT=scope.tar
 SCOPE_UI_BUILD_EXPORT=scope_ui_build.tar
 SCOPE_UI_BUILD_IMAGE=weave/scope-ui-build
 
-all: $(SCOPE_EXPORT)
+all: docker
+	
+docker: $(SCOPE_EXPORT)
```
Comment (tomwilkie) [inline Makefile]: Whats the point in this?

File: README.md [hunk 3]
```diff
@@ -13,33 +14,33 @@ Please note that the code, and especially the building and running story, is in
...
-```
-make client
-make static
-make build
-```
 
-Or, as a shortcut,
+## Build
+
+### Build in-place
```
Comment (tomwilkie) [inline README.md]: Should we include this?  We want to push everyone towards running on docker.  The fact that you guys develop on macs and want to run in place is just going to be confusing.

File: README.md [hunk 4]
```diff
@@ -47,24 +48,33 @@ To build a Docker container,
 make docker
```
Comment (tomwilkie) [inline README.md]: This should just be make

File: README.md [hunk 5]
```diff
@@ -47,24 +48,33 @@ To build a Docker container,
...
 make scope_ui_build.tar
 make static
 ```
 
-## Running
 
-### Manually
+## Run
+
+### Run in-place
```
Comment (tomwilkie) [inline README.md]: Ditto build-in-place.   Don't give the user options.  Its just confusing.

File: README.md [hunk 6]
```diff
@@ -47,24 +48,33 @@ To build a Docker container,
...
 
 1. Launch a probe process on each physical host you intend to monitor, via `sudo probe`
 2. Launch an app process, and configure it to talk to probes, via `app -probes="probe-host-1:4030,probe-host-2:4030"`.
 3. Load the user interface, via **http://app-host:4040**
 
-### As a Docker container
+### Run in a Docker container
```
Comment (tomwilkie) [inline README.md]: I don't think we need this.  The instruction above already cover this.

File: README.md [hunk 7]
```diff
@@ -47,24 +48,33 @@ To build a Docker container,
...
+
+Ensure Docker is running on your host, then,
+
+```
+./scope launch
+```
 
-TODO
+If you don't build the Docker container locally, `scope launch` will fetch a
```
Comment (tomwilkie) [inline README.md]: Maybe include this under the getting started instructions?

File: README.md [hunk 8]
```diff
@@ -13,33 +14,33 @@ Please note that the code, and especially the building and running story, is in
...
 
-To build the binaries in-place,
+To launch Scope inside of a Docker container,
 
 ```
-make build
+sudo wget -O /usr/local/bin/scope \
+  https://github.com/weaveworks/scope/releases/download/latest_release/scope
+sudo chmod a+x /usr/local/bin/scope
+sudo scope launch
```
Comment (tomwilkie) [inline README.md]: Have you tested this?  I hadn't got round to it yet (needed to have the images pushing to docker first).  That why I hadn't made a PR for them...

Comment (peterbourgon) [inline README.md]: Yep, it downloads but doesn't run, see #74

File: Makefile (hunk 2 above)
Comment (peterbourgon) [inline Makefile]: I struggle conceptually with the idea that a plain `make` target should depend on having a running Docker daemon. But I'm giving up on that.

File: README.md (hunk 6 above)
Comment (peterbourgon) [inline README.md]: Ack

File: README.md (hunk 7 above)
Comment (peterbourgon) [inline README.md]: Ack

**Comment by peterbourgon** [comment]
Changed as you suggested.

**Comment by tomwilkie** [comment]
LGTM; you might want to start a docs dir and put the stuff removed (about running locally for t&d) in there.

**Comment by tomwilkie** [comment]
Whilst you're doing this, you could stick the source graph badge in there:

[status](https://sourcegraph.com/github.com/weaveworks/scope)

**Pull request merged**